aptos/token_bridge/sources/bridge_state.move

```diff
@@ -159,10 +303,18 @@ module token_bridge::bridge_state {
         table::upsert(&mut state.bridge_implementations, chain_id, bridge_contract);
     }
 
-    public entry fun set_wrapped_asset(token_chain_id: U16, token_address: vector<u8>, wrapper: vector<u8>) acquires State {
+    public entry fun set_wrapped_asset(native_info: &OriginInfo, wrapper: vector<u8>) acquires State {
```

Comment:
did we want this `public` (and `entry`)? It looks like it allows anyone to override the state

Reply:
made these public(friend) and non-entry

Reply:
some state-modifying functions are split between `bridge_state` and `bridge_implementation`, which is why it's not totally private